src/hardware/input/mouse_config.cpp

```diff
@@ -459,24 +475,34 @@ static void config_init(Section_prop& secprop)
-	prop_bool->Set_help(
-	        "Enable the built-in DOS mouse driver ('on' by default). This results in the\n"
+	prop_str = secprop.Add_string("builtin_dos_mouse_driver",
+	                              only_at_start,
+	                              BuiltinDosDriverOn);
+	assert(prop_str);
+	prop_str->Set_values(
+	        {BuiltinDosDriverOff, BuiltinDosDriverOn, BuiltinDosDriverNoTsr});
+	prop_str->Set_help(
+	        "Built-in DOS mouse driver mode ('on' by default). This driver results in the\n"
 	        "lowest possible latency and the smoothest mouse movement, so only disable it\n"
 	        "and load a real DOS mouse driver if it's really necessary (e.g., if a game is\n"
 	        "not compatible with the built-in driver).\n"
-	        "  on:   Enable the built-in mouse driver. `ps2_mouse_model` and\n"
-	        "        `com_mouse_model` have no effect on the built-in driver.\n"
-	        "  off:  Disable the built-in mouse driver (if you don't want mouse support or\n"
-	        "        you want to load a real DOS mouse driver). To use a real DOS driver\n"
-	        "        (e.g., MOUSE.COM or CTMOUSE.EXE), configure the mouse type with\n"
-	        "        `ps2_mouse_model` or `com_mouse_model`, then load the driver.\n"
-	        "        A real DOS driver might increase compatibility with some programs,\n"
-	        "        but will introduce more input latency.\n"
-	        "Note: The built-in driver is auto-disabled if you boot into real MS-DOS or\n"
-	        "      Windows 9x under DOSBox. Under Windows 3.x, the driver is not disabled,\n"
-	        "      but the Windows 3.x mouse driver takes over.");
+	        "  on:      Simulate a mouse driver TSR program loaded from AUTOEXEC.BAT. This is\n"
+	        "           the most compatible way to emulate the DOS mouse driver, but if it\n"
+	        "           doesn't work with your game, try the 'no-tsr' setting.\n"
+	        "  no-tsr:  Enable the mouse driver without simulating the TSR program. Let us\n"
+	        "           know if it fixes any software not working with the 'on' setting.\n"
+	        "  off:     Disable the built-in mouse driver. It can still be started in runtime\n"
+	        "           by manually running the MOUSE.COM utility available on drive Z.\n"
+	        "Notes:\n"
+	        "  - The `ps2_mouse_model` and `com_mouse_model` settings have no effect on the\n"
+	        "    built-in driver.\n"
+	        "  - The driver is auto-disabled if you boot into real MS-DOS or Windows 9x\n"
+	        "    under DOSBox. Under Windows 3.x, the driver is not disabled, but the\n"
+	        "    Windows 3.x mouse driver takes over.\n"
+	        "  - To use a real DOS driver (e.g., MOUSE.COM or CTMOUSE.EXE), configure the\n"
+	        "    mouse type with `ps2_mouse_model` or `com_mouse_model`, then load the\n"
+	        "    driver. A real DOS driver might increase compatibility with some programs,\n"
+	        "    but will introduce more input latency.");
```

Comment:
Maybe include the technical info here about the built-in driver bypassing most of the DOS-side mouse layers.